cdap-security/src/test/java/co/cask/cdap/security/store/FileSecureStoreTest.java

```diff
@@ -120,22 +133,22 @@ public void testGetMetadata() throws IOException {
     Assert.assertEquals(metadata2.getName(), secureStore.getSecureData(NAMESPACE1, KEY2).getMetadata().getName());
   }
 
-  @Test(expected = IOException.class)
-  public void testOverwrite() throws IOException, InterruptedException {
+  @Test(expected = Exception.class)
+  public void testOverwrite() throws Exception {
     secureStoreManager.putSecureData(NAMESPACE1, KEY1, VALUE1.getBytes(Charsets.UTF_8), DESCRIPTION1, PROPERTIES_1);
     SecureStoreData oldData = secureStore.getSecureData(NAMESPACE1, KEY1);
     Assert.assertArrayEquals(VALUE1.getBytes(Charsets.UTF_8), oldData.get());
     String newVal = "New value";
     secureStoreManager.putSecureData(NAMESPACE1, KEY1, newVal.getBytes(Charsets.UTF_8), DESCRIPTION1, PROPERTIES_1);
   }
 
-  @Test(expected = IOException.class)
-  public void testGetNonExistent() throws IOException {
+  @Test(expected = Exception.class)
```

Comment:
NamespaceNotFoundException? Otherwise this test will pass even if some other exception is thrown?